logging/log_config.go

```diff
@@ -43,7 +44,7 @@ func buildRegexFromPattern(pattern string) string {
 	for _, ch := range pattern {
 		switch ch {
 		case '*':
-			matcher.WriteString(validLoggerSections)
+			matcher.WriteString(`.*`)
```

Comment:
Was this a bug? Why are we making this more permissive?

Reply:
yeah it was.`validLoggerSections` writes a regex to the string that doesn't match on `:` or `/` found in resource name loggers which caused them to never be configured. at this point `pattern` has already been validated to match as a legal pattern so its fine to match `*` any string now